src/commcare_cloud/commands/deploy/commcare.py

```diff
@@ -88,8 +88,9 @@ def deploy_commcare(environment, args, unknown_args):
             '--set', ','.join(fab_settings), branch=args.branch, *unknown_args
         )
     if rc != 0:
+        resume_option = color_success(f"--resume={environment.release_name}")
```

Comment:
nit: success doesn't feel like the right semantic category to me. Maybe `color_notice`?

Reply:
I was going for green, as in

"Deploy failed." (red, stop)  
"Add --resume=... to retry." (green, go)

I could use `clint.textui.colored.green` instead of `color_success`. I considered `color_notice`, but yellow often looks pretty washed out by default for folks who don't use a dark background.

Alternately, I could make it red. That would also stand out nicely.

Reply:
🤷 I think maybe if that's the problem (`color_notice` looks bad on a light background) we should change the color scheme. That must be a general annoyance for you anywhere `color_notice` is used. I'm not sure, but maybe there's a way to detect the terminal background color, and have one palate for a light background and another one for a dark background. Just throwing out ideas. Obviously very tangential to the purpose of this PR

Reply:
`color_notice` is fine. https://github.com/dimagi/commcare-cloud/pull/5849/commits/2e4d8d9c87406037612d342489be263457d37a2e

I can reconfigure my terminal to use a color that is more visible than the default yellow color (actually, I had forgotten that I did this long ago). It surprises me that the default yellow color is so bad against white. Maybe I'm the only person who doesn't like dark mode?